kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/engine/DefaultEngine.java

```diff
@@ -50,6 +52,12 @@ public ParquetHandler getParquetHandler() {
         return new DefaultParquetHandler(hadoopConf);
     }
 
+    @Override
+    public CommitCoordinatorClientHandler getCommitCoordinatorClientHandler(
+            String name, Map<String, String> conf) {
+        return new DefaultCommitCoordinatorClientHandler(hadoopConf, name, conf);
+    }
```

Comment:
Lets move out the default implementation out of the PR as the PR is just about the interfaces.